opportunity_job_api/src/apps/jobs/models/job_enrollment.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from apps.common.models import BaseModel
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from apps.jobs.models import JobOffer
+from apps.users.models.applicant_account import ApplicantAccount
+
+class JobEnrollment(BaseModel):
+
+    class Meta:
+        verbose_name = _("Job Enrollment")
+        verbose_name_plural = _("Job Enrollments")
+
+    job_offer = models.ForeignKey(JobOffer, null=True, on_delete=models.CASCADE)
```

Comment:
Stavi svaki argument u zaseban red.
Svaki argument treba da ima svoj key (prvom nedostaje `to`).
Klasu zameni sa string referencom.
Dodaj `verbose_name` argument.

Reply:
Ispravljeno.